docs/OnlyOffice.rst

```diff
@@ -0,0 +1,29 @@
+.. _OnlyOffice:
+
+#############################
+OnlyOffice Connector by eXo
+#############################
+
+**What is OnlyOffice**
+
+`OnlyOffice <https://www.onlyoffice.com/>`__ is a *partner editor service* 
+that allows you to create, edit  and collaborate online on your office 
+documents.
+
+|image0|
+
+.. note:: The supported documents types are: docx, pptx and xlsx.
+
+eXo OnlyOffice connector add-on is available only with the **commercial 
```

Comment:
```suggestion
eXo partnered with eXo to offer its customer the enterprise-ready version of OnlyOffice editors along with the **commercial
```

Reply:
Looks like a typing error I will change the second eXo by OnlyOffice online editors

Reply:
yes, my bad

Reply:
fixed and slightly reworked